### PR TITLE
feat(pace-184): add docker image for dbt module

### PR DESCRIPTION
**Pull request by trietsch**

### CODE REVIEW

File: dbt/src/main/kotlin/com/getstrm/pace/dbt/ModelWriter.kt [hunk 1]
```diff
@@ -24,7 +29,10 @@ class ModelWriter(private val policy: DataPolicy, private val sourceModel: DbtMo
      */
     @VisibleForTesting
     fun targetFilePath(target: DataPolicy.Target): String {
-        return sourceModel.originalFilePath.substringBeforeLast("/") +
+        return sourceModel.originalFilePath
+            // Windows uses backslashes, so we need to replace them with forward slashes
+            .replace("\\", "/")
+            .substringBeforeLast("/") +
```
Comment (bobvandenhoogen) [inline dbt/src/main/kotlin/com/getstrm/pace/dbt/ModelWriter.kt:35]: Does this also work when the jar is executed directly?

Comment (trietsch) [reply dbt/src/main/kotlin/com/getstrm/pace/dbt/ModelWriter.kt:35]: It should, as this is part of the manifest and therefore does not differ from executing via the Docker image

File: dbt/src/main/kotlin/com/getstrm/pace/dbt/ManifestParser.kt [hunk 2]
```diff
@@ -40,6 +40,7 @@ object ManifestParser {
         return modelNodes.filter {
             it.isPaceEnabled()
         }.map {
+            System.err.println("Creating policy for model ${it.originalFilePath}")
```
Comment (ivan-p92) [inline dbt/src/main/kotlin/com/getstrm/pace/dbt/ManifestParser.kt]: Why `err`?

File: dbt/src/main/kotlin/com/getstrm/pace/dbt/ModelWriter.kt [hunk 3]
```diff
@@ -11,10 +11,15 @@ class ModelWriter(private val policy: DataPolicy, private val sourceModel: DbtMo
...
-            file.writeText("$header\n$query\n")
-            println("Generated PACE model $targetFilePath")
+
+            try {
+                val file = File(targetFilePath)
+                val header = ModelHeaderRenderer(sourceModel, target).render()
+                file.writeText("$header\n$query\n")
+                println("Generated PACE model $targetFilePath")
+            } catch (e: Exception) {
+                println("Error writing model to $targetFilePath: $e")
```
Comment (ivan-p92) [inline dbt/src/main/kotlin/com/getstrm/pace/dbt/ModelWriter.kt]: Maybe `System.err` here?

**Review by ivan-p92 — APPROVED** [review]

File: dbt/src/main/kotlin/com/getstrm/pace/dbt/ManifestParser.kt (hunk 2 above)
Comment (trietsch) [reply dbt/src/main/kotlin/com/getstrm/pace/dbt/ManifestParser.kt]: Because I didn't want to interfere with the current stdout. Let's dicsuss.

**Pull request merged**